easy-ui-react/src/SearchNav/SearchNav.mdx

```diff
@@ -25,17 +25,27 @@ less than `640px`.
 
 <Canvas of={SearchNavStories.Simple} />
 
+## Emphasized Text
```

Comment:
**question**: do we want to update this to reflect the name change to title or keep it as-is?